Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!groovy
 
-@Library('github.com/mozmar/jenkins-pipeline@20170126.1')
+@Library('github.com/mozmar/jenkins-pipeline@15a2f1aeb08e58e2ee325c072b596d198771ccf9')
```

Comment:
There just wasn't a dated tag for the version I wanted yet. I can tag now and switch to it.